tests/contrib/django/test_django_dbm.py

```diff
@@ -0,0 +1,90 @@
+from django.db import connections
+import mock
+
+from ddtrace import Pin
+from tests.contrib import shared_tests
+from tests.utils import DummyTracer
+from tests.utils import override_config
+from tests.utils import override_dbm_config
+from tests.utils import override_env
+from tests.utils import override_global_config
+
+from ...contrib.config import POSTGRES_CONFIG
+
+
+def get_cursor(tracer, service=None, propagation_mode="service", tags={}):
+    conn = connections["postgres"]
+    POSTGRES_CONFIG["db"] = conn.settings_dict["NAME"]
+
+    cursor = conn.cursor()
+
+    pin = Pin.get_from(cursor)
+    assert pin is not None
+
+    pin.clone(tracer=tracer, tags={**pin.tags, **tags}).onto(cursor)
+
+    return cursor
+
+
+def test_django_postgres_dbm_propagation_enabled(tracer, transactional_db):
+    with override_dbm_config({"propagation_mode": "full"}):
```

Comment:
does this work?

the dbm event handlers should already be registered by now, so changing the config shouldn't impact that?

do you need an explicit call to `listen` ?

Reply:
yes this works as intended. the idea here is to:
- override the `dbm_config` object with the correct propagation
- call `listen` to enable the event handlers
- do test
- reset with `unlisten` so other tests aren't impacted by dbm propagation

Reply:
OH! I missed that `override_dbm_config` is new, I thought I saw it existing in another test somewhere